deployability/modules/setup.py

```diff
@@ -0,0 +1,42 @@
+
+# Copyright (C) 2015-2024, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

Comment:
Use only the 2015 year, for example https://github.com/wazuh/wazuh-packages/blob/master/bump_version.py#L1-L9

Also, this should be present in at least, all

Reply:
Done in [6941f08](https://github.com/wazuh/wazuh-qa/pull/4990/commits/6941f0899b9f8fea932167fe3aa91d484efa29c4)

Reply:
Added into all scripts in [0eb255a](https://github.com/wazuh/wazuh-qa/pull/4990/commits/0eb255ab91b41a72c7069491c04ef81565a4d195)